### PR TITLE
Add bullets

### PR DESCRIPTION
**Pull request by urpylka**
Hi, I added bullets to the doc to make it more readable.

### Description
Minor changes - is nothing critical.

### Checklist

This pull request is:

- [x] A documentation / typographical / small typing error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

### CODE REVIEW

**Review by zzzeek — APPROVED** [review]

File: doc/build/orm/inheritance.rst [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
-up among dependent tables, each class represented by its own table that only
+SQLAlchemy supports three forms of inheritance:
+
+* **single table inheritance** – several types of classes are represented by a single table;
+
+* **concrete table inheritance** – each type of class is represented by independent tables;
+
+* **joined table inheritance** – the class hierarchy is broken up among dependent tables.
+
+Each class represented by its own table that only
```
Comment (CaselIT) [inline doc/build/orm/inheritance.rst:14]: I think this part and the phrase below refers to the "joined table inheritance" section

**Review by CaselIT — CHANGES_REQUESTED** [review]
can you update one of the bullets as suggested?

**Comment by urpylka** [comment]
> can you update one of the bullets as suggested?

Done

**Comment by CaselIT** [comment]
thanks!

**Review by CaselIT — APPROVED** [review]

**Pull request merged**